website/docs/docs/build/udfs.md

```diff
@@ -0,0 +1,110 @@
+---
+title: "User-defined functions"
+description: "Learn how to define UDFs in your dbt projects."
+id: "udfs"
+---
+
+## Overview
+
+User-defined functions (UDFs) enable users to define and register custom functions within the warehouse. Like [macros](/docs/build/jinja-macros), UDFs enable reuse of code; however, unlike macros, you can define UDFs in languages other than SQL (for example, Python, Java, Scala) and you can use them in queries outside <Constant name="core" />.
+
+Note that only basic SQL UDFs are currently supported in <Constant name="core" />.
```

Comment:
I'm not sure if there's a better way to phrase this, but it feels contradictory to the part that immediately precedes it, where we say that they can define UDFs in languages other than SQL.